File: _study/COS_PRO_2/c/stage4/4.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int classes[], int classes_len, int m) {
    int answer = 0;
    for(int i=0; i<classes_len; i++) {
        answer += classes[i] / m;
        if (classes[i] % m != 0)
            answer++;
    }
    return answer;
}

int main() {
    int classes[] = {80, 45, 33, 20};
    int m = 30;
    int ret = solution(classes, 4, m);

    printf("solution 함수의 반환 값은 %d 입니다.\n", ret);
}

File: _study/COS_PRO_1/python/stage1/4.py
def solution(num):
    num += 1
    str_num = str(num)[::-1]
    j, k = 0, 1
    for i in str_num:
        if i == '0':
            j += k
            k *= 10
    answer = num + j
    return answer


num = 9949999;
ret = solution(num)

print("solution 함수의 반환 값은", ret, "입니다.")
